releasenotes/notes/snmp-traps-resolve-enums-e66ef9e51a9269aa.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Resolve SNMP trap variables with integer enumerations to their string representation.
+issues:
+  - |
+    Enumerations for the BITS type are not yet supported.
```

Comment:
```suggestion
```
Let's not have an `issues` section for a feature that's not yet implemented

Reply:
👍